.infra/rdev/values.yaml

```diff
@@ -0,0 +1,15 @@
+###### Generated using `argus bootstrap` command ######
+stack:
+  services:
+    aws-oidc:
+      image:
+        tag: sha-8700083
+      env:
+        - name: OKTA_CLIENT_ID
+          value: aws-config
+        - name: OKTA_ISSUER_URL
+          value: https://czi-prod.okta.com
+        - name: OKTA_SERVICE_CLIENT_ID
+          value: 0oa2m0fqfwCAASN9K5d7
```

Comment:
Should we try and use argus secrets for these?

Reply:
I try to check in everything that isn't a secret so its more easily found. Since these weren't secrets, I just committed them. The actual private key I stored in argus secrets